docs/Guides/Deprecations.md

```diff
@@ -0,0 +1,41 @@
+<h1 align="center">Fastify</h1>
+
+# Deprecations
+
+It is suggested that if you are deprecating features, 
```

Comment:
This documentation is facing the users of fastify and not the developers of fastify.